docker-compose.yaml

```diff
@@ -0,0 +1,35 @@
+name: lerobot
+
+services:
+  lerobot:
```

Comment:
In future PR's we could split up this service to be `lerobot-x86` and `lerobot-arm` to support multiple platforms (like jetson).

For now, I think let's see how people use this, improve the experience, then tackle multi-arch support in the future.